src/common/components/CustomIcon/SunIcon.tsx

```diff
@@ -0,0 +1,12 @@
+import { CustomIcon, type CustomIconProps } from "./CustomIcon";
+
+export const SunIcon = (props: CustomIconProps) => {
+  return (
+    <CustomIcon width="1em" height="1em" viewBox="0 0 256 256" {...props}>

```

Comment:
question: is there a reason to use `em` here instead of `px`, also not sure if there will be scaling issues if using "0 0 256 256" instead of "0 0 24 24" (not sure if figma has a 0 0 24 24 sun and moon icon). as of now the `ThemeToggle` component lgtm! but not sure if we try to add `size` prop to it if we ever use these icons elsewhere in the future

Reply:
nice catch! didn't notice this as I exported it from a third-party site

re-uploaded another one from figma here - 1e312e7